core/payment/src/processor.rs

```diff
@@ -388,7 +388,7 @@ impl PaymentProcessor {
             activity_payment.allocation_id = None;
         }
 
-        counter!("payment.amount.sent", ya_metrics::utils::cryptocurrency_to_u64(&msg.amount), "driver" => msg.driver);
+        counter!("payment.amount.sent", ya_metrics::utils::cryptocurrency_to_u64(&msg.amount), "platform" => msg.driver);
```

Comment:
The value should be `payment_platform` not `msg.driver` then.

Reply:
Thanks for noticing

Reply:
Now you're using a moved value. Compile before committing. :P

Reply:
Eh. I hoped it will be easy fix :P